raddb/sites-available/tls

```diff
@@ -515,41 +523,42 @@ home_server tls {
 		#    'c_rehash' is OpenSSL's command.
 		#  3) uncomment the line below.
 		#  5) Restart radiusd
-	#	check_crl = yes
+#		check_crl = yes
 		ca_path = ${cadir}
 
-	       #
-	       #  If check_cert_issuer is set, the value will
-	       #  be checked against the DN of the issuer in
-	       #  the client certificate.  If the values do not
-	       #  match, the certificate verification will fail,
-	       #  rejecting the user.
-	       #
-	       #  In 2.1.10 and later, this check can be done
-	       #  more generally by checking the value of the
-	       #  TLS-Client-Cert-Issuer attribute.  This check
-	       #  can be done via any mechanism you choose.
-	       #
-	#       check_cert_issuer = "/C=GB/ST=Berkshire/L=Newbury/O=My Company Ltd"
-
-	       #
-	       #  If check_cert_cn is set, the value will
-	       #  be xlat'ed and checked against the CN
-	       #  in the client certificate.  If the values
-	       #  do not match, the certificate verification
-	       #  will fail rejecting the user.
-	       #
-	       #  This check is done only if the previous
-	       #  "check_cert_issuer" is not set, or if
-	       #  the check succeeds.
-	       #
-	       #  In 2.1.10 and later, this check can be done
-	       #  more generally by checking the value of the
-	       #  TLS-Client-Cert-Common-Name attribute.  This check
-	       #  can be done via any mechanism you choose.
-	       #
-	#	check_cert_cn = %{User-Name}
-	#
+		#
```

Comment:
can you redo this without the whitespace changes?  That makes it harder to see which changes are important, and which are just whitespace.

Reply:
Done @alandekok